### PR TITLE
add Broadway v0.1.0 w/ git auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/mbebenita/Broadway
**Watch 89
  Star 1,290
 Fork 180**
@PeterDaveHello  could u help me check this PR for #5706  ? thank you!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [ ] No already exist / duplicated issue and PR https://github.com/cdnjs/cdnjs/pull/6044
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Have you confirmed with its author about what are the files should be hosted ?

**Comment by maruilian11** [comment]
@PeterDaveHello not yet, i have done it here
https://github.com/mbebenita/Broadway/issues/123

**Comment by PeterDaveHello** [comment]
Any updates?

**Comment by maruilian11** [comment]
@PeterDaveHello the issue still doesn't get response from the author, can I add all js files under `Player`(https://github.com/cdnjs/cdnjs/issues/5706#issuecomment-151176403) and rebase this branch with our latest master branch?

**Comment by PeterDaveHello** [comment]
Okay let's do it.

**Comment by maruilian11** [comment]
@Piicksarn @cdnjs/intern2 could you please help me review this PR? Thanks!

File: ajax/libs/Broadway/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "description": "A JavaScript H.264 decoder.",
+  "author": "Michael Bebenita",
+  "keywords": [
+    "decoder",
+    "JavaScript",
+    "H.264",
+    "player"
+  ],
+  "licenses": [
+    "Apache-2.0",
```
Comment (PeterDaveHello) [inline ajax/libs/Broadway/package.json]: What's the reference?

**Comment by maruilian11** [comment]
@pvnr0082t thanks for your review, I have remove `screen.css` and turn filemap to `!(screen*).+(js|css|map)` which has been tested.
And `Apache-2.0` is for cpp file under Android project, I mistook before.

File: ajax/libs/Broadway/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  "version": "0.1.0",
+  "description": "A JavaScript H.264 decoder.",
+  "author": "Michael Bebenita",
+  "keywords": [
+    "decoder",
+    "JavaScript",
+    "H.264",
+    "player"
+  ],
+  "licenses": [
```
Comment (pvnr0082t) [inline ajax/libs/Broadway/package.json]: Please use `license` instead of `licenses`. Thanks.

**Comment by maruilian11** [comment]
@pvnr0082t thank you! how stupid mistake I made...

File: ajax/libs/Broadway/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "author": "Michael Bebenita",
+  "keywords": [
+    "decoder",
+    "JavaScript",
+    "H.264",
+    "player"
+  ],
+  "license": [
+    "BSD-3-Clause"
+  ],
```
Comment (pvnr0082t) [inline ajax/libs/Broadway/package.json]: Let's use `"license": "BSD-3-Clause",` here.

**Comment by maruilian11** [comment]
wait for https://github.com/mbebenita/Broadway/issues/123#issuecomment-263466273

**Comment by BlueT** [comment]
Expecting for updates

**Comment by maruilian11** [comment]
@PeterDaveHello @cdnjs/intern2 If there is no response for days, can we use [the current git log commit](https://github.com/mbebenita/Broadway/commit/d28fd6328a299e1bfc8a883bf2b94b7e53a2b77e) number as version to add it first in this case?

**Comment by PeterDaveHello** [comment]
I don't want to use commit hash as version anymore, let's just add the tagged version.
@kennynaoh @pvnr0082t please help review this PR, thanks.

**Comment by x09326** [comment]
@maruilian11 Please rebase to the latest master branch, thanks.

**Comment by BlueT** [comment]
Looks like that @PeterDaveHello 's change request has been gone, but still blocks the auto merge process.
Also need one more LGTM.

**Comment by PeterDaveHello** [comment]
@BlueT We'll get this merged asap, thanks!

**Pull request merged**